src/checkout_body/checkout_body.css

```
@@ -0,0 +1,207 @@
+.checkout_section {
+    padding: 86px 174px 44px 175px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.checkout_header {
+    font-size: 36px;
+    font-weight: 600;
+    letter-spacing: .5px;
+}
+
+.checkout_form {
+    padding-top: 36px;
+}
+
+.double_iputs_container {
+    display: flex;
+    gap: 31px;
+}
+
+.standart_chek_input_container {
+    display: flex;
+    flex-direction: column;
+    padding-top: 36px;
+}
+
+.double_iputs_item {
+    display: flex;
+    flex-direction: column;
+}
+
+.inputs_label {
+    font-size: 16px;
+    font-weight: 600;
+    padding-bottom: 22px;
+}
+
+.double_checkkout_input {
+    width: 211px;
+    height: 75px;
+}
+
+
+.standart_chek_input {
+    width: 453px;
+    height: 75px;
+}
+
+.check_input {
+    font-size: 16px;
+    font-weight: 400;
+    border-radius: 10px;
+    border: 1px solid #9F9F9F;
+    background: #FFF;
+}
+
+.checkout_selector {
+    color: #9F9F9F;
+    width: 453px;
+    height: 75px;
+    padding-left: 25px;
+    position: relative;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    background-image: url('/public/photos/64.svg');
+    background-repeat: no-repeat;
+    background-position: right 26px center;
+}
+
+.standart_chek_input::placeholder {
+    padding-left: 30px;
+}
+
+.checkout_product_container {
+    display: flex;
+    flex-direction: column;
+    width: 531px;
+}
+
+.checkout_product_item {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+}
+
+.checkout_product_header {
+    font-size: 24px;
+    font-weight: 600;
+    padding-bottom: 15px;
+}
+
+.checkout_product_text {
+    padding-top: 22px;
+    font-size: 16px;
+    font-weight: 500;
+}
+
+.checkout_product_quant {
+    padding-left: 10px;
+    color: #000;
+    font-size: 12px;
+    font-weight: 500;
+}
+
+.checkout_product_name {
+    color: #9F9F9F;
+    font-size: 16px;
+    font-weight: 400;
+}
+
+.checkout_product_cost {
+    font-size: 16px;
+    font-weight: 300;
+}
+
+.place_order {
+    margin: 51px 0px 0px 144px;
+    width: 527px;
+}
+
+.checkout_product_total_cost {
+    color: var(--primary, #B88E2F);
+    font-size: 24px;
+    font-weight: 700;
+    padding-top: 11px;
+}
+
+.checkout_place_order {
+    margin-top: 35px;
+    padding-top: 23px;
+    border-top: 2px solid #D9D9D9;
+    display: flex;
+    flex-direction: column;
+}
+
+.make_sure_directly {
+    padding-top: 11px;
+    padding-bottom: 25px;
+    color: #9F9F9F;
+    text-align: justify;
+    font-size: 16px;
+    font-weight: 300;
+}
+
+.personal_data_support {
+    text-align: justify;
+    font-size: 16px;
+    font-weight: 300;
+}
+
+.direct_inp {
```

```suggestion
.direct_input {
```
do not shorten the words